map/src/map/layers/LocalClientTrackLayer.js

```diff
@@ -434,9 +441,9 @@ export default function LocalClientTrackLayer() {
         layers.addLayer(marker);
     }
 
-    function createLocalTrack(file, points) {
+    function createLocalTrack(file, points = [], wpts = []) {
         TracksManager.prepareTrack(file);
-        file.tracks = [{points: points}];
+        file.tracks = [{points: points, wpts: wpts}];
```

Comment:
formatting

Reply:
please post examples of correct formatting for this line

Reply:
file.tracks = [
    {
        points: points, 
        wpts: wpts
    }
];

Reply:
<img width="194" alt="Screenshot 2023-06-30 at 08 43 29" src="https://github.com/osmandapp/web/assets/38911173/b9f7a24b-c8d8-4349-9e73-fd08f2c4fdfb">